device-registry/src/main/scala/org/genivi/sota/device_registry/GroupMember.scala

```diff
@@ -40,4 +40,9 @@ object GroupMember {
       .filter(r => r.groupId === groupId)
       .map(r => r.deviceUuid)
       .result
+
+  def countDevicesInGroup(groupId: Uuid)(implicit ec: ExecutionContext): DBIO[Int] =
+    GroupInfoRepository.getGroupInfoById(groupId)
```

Comment:
Can you indent before the .getGroupInfoById()? Then it matches the rest of the file